specification/common-types/resource-management/v5/types.json

```diff
@@ -47,23 +88,39 @@
           "description": "Indicates if the resource name is available."
         },
         "reason": {
-          "type": "string",
-          "description": "The reason why the given name is not available.",
-          "enum": [
-            "Invalid",
-            "AlreadyExists"
-          ],
-          "x-ms-enum": {
-            "name": "CheckNameAvailabilityReason",
-            "modelAsString": true
-          }
+          "$ref": "#/definitions/CheckNameAvailabilityReason",
+          "description": "The reason why the given name is not available."
         },
         "message": {
           "type": "string",
-          "description": "Detailed reason why the given name is available."
+          "description": "Detailed reason why the given name is not available."
         }
       }
     },
+    "EncryptionStatus": {
+      "type": "string",
+      "description": "Indicates whether or not the encryption is enabled for container registry.",
```

Comment:
>for container registry [](http://example.com/codeflow?start=73&length=22)

this description text "for container registry" looks too specific, for common types!

Reply:
in this particular case we took this description: https://github.com/Azure/azure-rest-api-specs/blob/4a748a953cb8408283e3b5e9fd8773e012ca74ae/specification/common-types/resource-management/v5/types.json#L535

Reply:
I guess we can fix it later...